osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/formValidation.js

```diff
@@ -68,34 +60,40 @@
-                    this.setSubmitButtonState();
-                    
-                    this.excerptBoundInputFields.forEach(function(excerptBoundInputField) {
-                        if(this.validity['excerptBounds']) {
-                            excerptBoundInputField.setCustomValidity('');
-                            document.getElementById('excerpt-validation').textContent = '';
-                        } else {
-                            var howMuchTooLarge = estimatedFileSize ? Math.ceil(estimatedFileSize * 100 / allowedMaxSize - 100) + '% ' : '';
-                            var message = 'Excerpt {percent}too large!'.replace('{percent}', howMuchTooLarge);
-                            excerptBoundInputField.setCustomValidity(message);
-                            document.getElementById('excerpt-validation').textContent = message;
-                        }
-                    }.bind(this));
-                }.bind(this)
-            );
+            if(this.formModeSwitcher.value == "new-excerpt") {
+                var allowedMaxSize = 250 * 1024 * 1024;
+
+                var north = document.getElementById('id_north').value;
+                var east = document.getElementById('id_east').value;
+                var south = document.getElementById('id_south').value;
+                var west = document.getElementById('id_west').value;
+
+                jQuery.getJSON(
+                    '/api/estimated_file_size/',
+                    {'north': north, 'east': east, 'south': south, 'west': west},
+                    function (data) {
+                        var estimatedFileSize = Number(data['estimated_file_size_in_bytes']);
+                        this.validity['excerptBounds'] = estimatedFileSize < allowedMaxSize;
+                        this.setSubmitButtonState();
+
+                        this.excerptBoundInputFields.forEach(function (excerptBoundInputField) {
+                            if (this.validity['excerptBounds']) {
+                                excerptBoundInputField.setCustomValidity('');
+                                document.getElementById('excerpt-validation').textContent = '';
+                            } else {
+                                var howMuchTooLarge = estimatedFileSize ? Math.ceil(estimatedFileSize * 100 / allowedMaxSize - 100) + '% ' : '';
+                                var message = 'Excerpt {percent}too large!'.replace('{percent}', howMuchTooLarge);
+                                excerptBoundInputField.setCustomValidity(message);
+                                document.getElementById('excerpt-validation').textContent = message;
+                            }
+                        }.bind(this));
+                    }.bind(this)
+                );
+            } else {
+                this.validity['excerptBounds'] = true;
+                document.getElementById('excerpt-validation').textContent = '';
```

Comment:
Not changed in this PR, but:

Can the ID of this element be changed to something more descriptive, e.g. `'excerpt-validation-message'` or `'excerpt-validation-result'`?

Also, some ID values contain an `'id_'` prefix, some don't. Is that intentional?